cmd/gbounty/bootstrap/bootstrap.go

```diff
@@ -367,7 +381,7 @@ func runScan(
 						Occurrences:           occ,
 						ProfileType:           prof.GetType().String(),
 						At:                    time.Now().UTC(),
-					}, cfg.ShowResponses,
```

Comment:
If you follow the approach I suggested here https://github.com/BountySecurity/gbounty/pull/8/files#r1823608034, you won't need to pass this in the call tree.